kolibri/plugins/learn/assets/src/views/content-card/card.styl

```diff
@@ -5,4 +5,4 @@ $thumb-width-desktop = 210px
 $thumb-height-desktop = round($thumb-width-desktop / $ratio)
 
 $thumb-width-mobile = 150px
-$thumb-height-mobile = round($thumb-width-mobile / $ratio)
+$thumb-height-mobile = max(round($thumb-width-mobile / $ratio), 92px)
```

Comment:
This is going to make the thumbnails a little bit off the target `$ratio`

If the goal is to set a particular height of `92px`, you could do something like

```stylus
$thumb-height-mobile = 92px
$thumb-width-mobile = round($thumb-height-mobile * $ratio)
```